releases/R2023b/README.md

```diff
@@ -44,7 +44,7 @@ To deploy MATLAB Production Server onto an existing VPC, specify these additiona
 |--------------- | ----- |
 ||**Network License Manager**|
 | **Deploy License Server** | Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>If you are using an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
-|**IP Address and port number of Existing Network License Manager**|<p> Optional parameter: Specify the port number and private DNS name or private IP address of the network license manager that has already been deployed to the existing VPC. Specify it in the format <port>@<privateDNSname>, for example, 27000@ip-172-30-1-89.ec2.internal or 27000@172.30.1.89. By default, the license manager uses port 27000. Leave this field blank if you are deploying a new network license manager.</p>|
+|**IP Address and port number of Existing Network License Manager**|<p> Optional parameter: Specify the port number and private DNS name or private IP address of the network license manager that has already been deployed to the existing VPC. Specify it in the format <port>@<privateDNSname>, for example, `27000@ip-172-30-1-89.ec2.internal` or `27000@172.30.1.89`. By default, the license manager uses port 27000. Leave this field blank if you are deploying a new network license manager.</p>|
```

Comment:
27000@ip-172-30-1-89.ec2.internal no longer appears as a hyperlink but I noticed in the Readme that the words port and privateDNSname do not show up at all. I just see '@'. Maybe you could italicize port@privateDNSname and remove <>.